frontend/src/lib/utils/event-property-utls.tsx

```diff
@@ -0,0 +1,62 @@
+import { Tooltip } from 'lib/lemon-ui/Tooltip'
+
+import { ElementType } from '~/types'
+
+export function autocaptureToImage(
+    elements: ElementType[]
+): null | { src: string | undefined; width: string | undefined; height: string | undefined } {
+    const find = elements.find((el) => el.tag_name === 'img')
+    const image = {
+        src: find?.attributes?.attr__src,
+        width: find?.attributes?.attr__width,
+        height: find?.attributes?.attr__height,
+    }
+    return image.src ? image : null
+}
+
+export function AutocaptureImageTab({ elements }: { elements: ElementType[] }): JSX.Element | null {
+    const img = autocaptureToImage(elements)
+    if (img) {
+        return (
+            <div className="flex bg-bg-3000 items-center justify-center relative border-2">
+                {/* Transparent grid background */}
+                <div className="ImagePreview__background absolute h-full w-full" />
```

Comment:
Not sure I love the transparent PNG style background but don't really have a suitable alternative. Did you consider just making it the default background color?

Reply:
Yeah, it's because the first image I rendered this way had tall transparency and I though my render code was wrong. So I need to show the transparent part... but this is wrong... the transparent background should only be the bounds of the image so you can see where that is

Reply:
![Screenshot 2024-12-02 at 12 04 31](https://github.com/user-attachments/assets/ca49799d-ed34-4a81-aefd-1679f42090d6)

![Screenshot 2024-12-02 at 12 09 42](https://github.com/user-attachments/assets/cfb623a2-4730-41aa-86f3-1e93adb6a97f)

![Screenshot 2024-12-02 at 12 10 19](https://github.com/user-attachments/assets/455ff7bc-7c49-476d-a703-08979a54e0cf)
![Screenshot 2024-12-02 at 12 10 29](https://github.com/user-attachments/assets/dd9e1341-722b-4d2f-829f-48e9903d218e)

Reply:
so, we now show transparency _only_ where there is transparency
🙈